mock-core-configs/mock-core-configs.spec

```diff
@@ -59,6 +59,12 @@ HOST="fedora-%{fedora}"
 HOST="rhel-%{rhel}"
 %endif
 
+bootstrap_image_ON()
+{
+    echo >&2 "Turning bootstrap image ON for $1"
+    echo "config_opts['use_bootstrap_image'] = True" >> "$1"
```

Comment:
why not simple STDOUT?

Reply:
I later realized `set -x` is in action, there's no need to create additional traffic on stdout (I'll drop the method).